src/main/kotlin/bloombox/client/services/shop/ShopClient.kt

```diff
@@ -110,15 +116,14 @@ class ShopClient(override val host: String,
   /**
    * Channel for client->server traffic.
    */
-  override val channel: ManagedChannel = NettyChannelBuilder
-        .forAddress(host, port)
-        .executor(executor)
-        .sslContext(GrpcSslContexts.forClient()
-              .trustManager(this.javaClass.getResourceAsStream(authorityRoots))
-              .build())
-        .negotiationType(NegotiationType.TLS)
-        .intercept(interceptor)
-        .build()
+  override val channel: ManagedChannel = channelBuilder(
+        host = host,
+        port = port,
+        executor = executor,
+        clientAuth = clientAuth,
+        transportMode = transportMode,
+        clientCredentials = clientCredentials,
+        clientAuthorityRoots = clientAuthorityRoots).intercept(interceptor).build()
```

Comment:
new syntax for creating a client-side `ManagedChannel`, with support for crypto assets